panopticon/datadog.py

```diff
@@ -189,25 +189,34 @@ def wrapped_func(*args, **kwargs):
 
         return track_time_decorator
 
-    @staticmethod
-    def _convert_tags(tags):
+    @classmethod
+    def _convert_tags(cls, tags):
         """
         Convert tags, which may be a dict or iterable, into
         the DataDog format of a list of 'key:value' strings.
 
+        To make it easier to write test assertions that validate
+        mock calls, if the stats object has been mocked, the
+        resulting list is sorted.
+
         Args:
             tags (Dict, Sequence):
 
         Returns:
             [str]
         """
         if isinstance(tags, dict):
-            return [
+            result = [
                 '{}:{}'.format(key, value)
                 for key, value in tags.items()
             ]
+        else:
+            result = list(tags)
+
+        if isinstance(cls._stats_instance, mock.Mock):
+            return sorted(result)
```

Comment:
I guess it doesn't... I thought it might make sense to avoid the tiny performance hit. But I think you're right - I'll make it always sort. Fewer differences between tests and prod.